api/.env

```diff
@@ -81,3 +81,5 @@ MAIL_FROM="info@ecamp3.ch"
 # https://www.google.com/recaptcha/admin
 RECAPTCHA_SECRET="disabled"
 ###< google/recaptcha ###
+
+TRANSLATE_ERRORS_TO_LOCALES="en,en_CH_scout,de,de_CH_scout,fr,fr_CH_scout,it,it_CH_scout"
```

Comment:
Not sure if we need the scout versions at this stage already. I'd expect validation translation messages to be pretty generic, at least at the beginning.

Reply:
done